ArcGISRuntimeSDKQt_QMLSamples/Geometry/ClipGeometry/main.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <QSettings>
+#include <QGuiApplication>
+#include <QQuickView>
+#include <QCommandLineParser>
+#include <QDir>
+#include <QQmlEngine>
+
+#define STRINGIZE(x) #x
+#define QUOTE(x) STRINGIZE(x)
+
+int main(int argc, char *argv[])
+{
+
+  QGuiApplication app(argc, argv);
+  app.setApplicationName(QStringLiteral("%{Clip geometry} - QML"));
```

Comment:
`Clip Geometry - QML`

Reply:
Please remove the %{}. Please use the consistent naming per Lucas' previous instructions for both samples (Geometry or geometry). Be consistent with both, and I don't think the % and brackets are needed.